### DISCUSSION TITLE
How to deploy Supabase on Azure to achieve best performance?

### DISCUSSION THREAD

**Discussion by Sulomus** [supabase/supabase#26654] [Questions]
Hello,
I read several discussions on self-hosting supabase on Azure: 
1) https://www.reddit.com/r/Supabase/comments/1bowfi9/experience_with_managed_postgres_on_azure/
2) https://github.com/orgs/supabase/discussions/5112

Also I found that Bitnami can be used to self-host suapabse via Azure marketplace: https://bitnami.com/stack/supabase/cloud/azure

_Does anyone have any experience in self-hosting supabase on Azure? Is it worth it? And if so - how we can deploy Supabase on azure so that it can achieve good performance (in my amateur mindset I do not recognize hosting all in one VPS as a optimized setup)_

**Comment by TalkSimple** [comment]
I have the same question. Did you ever get an answer?

**Reply by anderpru** [reply]
did you found?

**Reply by TalkSimple** [reply]
Sadly no. However, I am on the supabase cloud platform and very happy
